main.go

```diff
@@ -1,28 +1,11 @@
 package main
 
 import (
-	"fmt"
-	"github.com/kartikx/obsidian-finances-parser/parser"
+	"github.com/kartikx/obsidian-finances-parser/pkg/cmd"
 )
 
 func main() {
-	expenses, err := parser.ParseStatement(FinanceStatementFilePath, FinanceStatementFormat)
-
-	if err != nil {
-		fmt.Println("Parsing Statement failed", err)
-		return
-	}
-
-	formattedExpenses, err := formatExpensesForObsidian(expenses)
-
-	if err != nil {
-		fmt.Println("Formatting expenses failed", err)
-		return
-	}
-
-	if len(OutputFilePath) > 0 {
-		writeToObsidianVault(formattedExpenses, OutputFilePath)
-	} else {
-		writeToConsole(formattedExpenses)
-	}
+	// run obsidian-finances-parser
+	obsidianParserCommand := cmd.NewObsidianParserCommand()
```

Comment:
How do we pass in the -input and -output flags here? As users, we can do:
```
ofp -input="input" -output="output"
```

But how do we do this in development?

Reply:
Originally the filepaths for input and output were hardcoded in `FinanceStatementFilePath` and `OutputFilePath`. Now, not passing the flags to the binary simply makes use of these hardcoded values when the binary is run - as it did previously.

These path strings are passed in as default values when the flags are created [here](https://github.com/renormalize/obsidian-finances-parser/blob/8a849afe02ae424ed2daa8209b8f7125365ba1ba/pkg/cmd/root.go#L41-L42).

Does this answer your question?

Reply:
Sounds good.